qa/os/src/test/java/org/elasticsearch/packaging/test/DockerTests.java

```diff
@@ -177,6 +180,30 @@ public void test70BindMountCustomPathConfAndJvmOptions() throws Exception {
         assertThat(nodesResponse, containsString("\"using_compressed_ordinary_object_pointers\":\"false\""));
     }
 
+    /**
+     * Check that the default config can be overridden using a bind mount, and that env vars are respected
+     */
+    public void test71BindMountCustomPathWithDifferentUID() throws Exception {
+        copyFromContainer(installation.config("elasticsearch.yml"), tempDir.resolve("elasticsearch.yml"));
+
+        final Path tempDataDir = mktempDir(getTempDir());
+        final Path tempEsDataDir = tempDataDir.resolve("esDataDir");
+        // Make the local directory and contents accessible when bind-mounted
+        mkDirWithPrivilegeEscalation(tempEsDataDir, 1500, 0);
+
+        // Restart the container
+        final Map<Path, Path> volumes = Map.of(tempEsDataDir.toAbsolutePath(), Path.of("/usr/share/elasticsearch/data"));
```

Comment:
I think you can do:

```suggestion
        final Map<Path, Path> volumes = Map.of(tempEsDataDir.toAbsolutePath(), installation.data);
```

Reply:
Thanks, addressed in d89f795d61b9e645bb903aa9c2693ec18838b18e